package.json

```diff
@@ -9,7 +9,6 @@
   "workspaces": {
     "packages": [
       "packages/blade",
-      "packages/razorsharp",
```

Comment:
once we remove this all the eslint and everything needs to be setup inside razorsharp then isn't it?